### ISSUE TITLE
Publicize: Add alt text for blank image

### ISSUE THREAD

**Issue by danjjohnson** [Automattic/jetpack#18307]
If a post contains an image with an alt attribute, that will be used for the `og:image:alt` tag for publicize.

But if there is no image, and the default blank.jpg is used, no alt text is provided, which can be confusing for blind people who won't be able to hear a description of the blank image.

#### Describe the solution you'd like

Add an `og:image:alt` tag for the blank.jpg image.

#### Additional context

Reported here: https://wordpress.org/support/topic/accessibility-of-posts-publicized-to-facebook/

**Comment by jenhooks** [comment]
@jeherve -- the user has provided some more information. According to them, screen readers cannot do anything with what Jetpack's Publicize pushes to Facebook:

> The issue is that Jetpack is not sending the Title, the content or the link to Facebook in an accesible fashion. 

Is this a Facebook issue, or a Publicize issue? I've asked the user to advise on how their screen reader handles [this page](https://www.facebook.com/accessibility).

**Comment by jeherve** [comment]
Ah, that's interesting. That would be an issue on Facebook's end. We can and do pass information to Facebook about a post's title, description, image, and more. The actual display of that information is something outside of our control; Facebook decides how to display post previews on its platform.

We can, and should, pass extra information about images when we can, so I think this GitHub issue remains valid. But if Facebook doesn't render an accessible post preview based on the information sent to them, that is something that should be reported to Facebook directly, here:
https://developers.facebook.com/support/bugs/
